src/write.jl

```diff
@@ -26,17 +26,45 @@ end
 Write out a `.pomdp` file using the POMDPs.jl interface
 Specification: http://cs.brown.edu/research/ai/pomdp/examples/pomdp-file-spec.html
 """
-function Base.write(io::IO, pomdp::POMDP)
+function Base.write(io::IO, pomdp::POMDP; pretty = false)
 
     ss = states(pomdp)
     as = actions(pomdp)
     os = observations(pomdp)
 	println(io, "discount: ", discount(pomdp))
 	println(io, "values: reward") # NOTE(tim): POMDPs.jl assumes rewards rather than costs
-    println(io, "states: ", length(ss))
-    println(io, "actions: ", length(as))
-    println(io, "observations: ", length(os), "\n")
-	
+
+	n_states = length(ss)
+	if pretty && n_states < 20
+    	println(io, "states: ", join([string(s) for s=ss], " "))
+	else
+    	println(io, "states: ", length(ss))
+	end
+
+	n_actions = length(as)
+	if pretty && n_actions < 20
```

Comment:
The silent `n_actions < 20` check is not good practice. The user should be allowed to do it with 21 actions if they want. At minimum, if an automatic "safety" feature like this is added it should at least notify the user with a message if it happens.

Reply:
Agreed. I wasn't sure how to implement this. Do you think this ought to go in the `POMDPFile` struct or something similar?

Reply:
I would recommend just letting people do it with however many they want. They are the ones who will have to suffer the consequences of extremely long lines in their pomdp files!

Reply:
I don't see any side effects for anyone else if they want to do it.

Reply:
I've opted to remove this. Perhaps I can leave a TODO to implement some kinda `POMDPFile` struct that contains this data, but in retrospect, people can pick their poison. 😅